zeppelin-interpreter/src/main/thrift/RemoteInterpreterService.thrift

```diff
@@ -56,6 +56,17 @@ struct RemoteInterpreterEvent {
   2: string data      // json serialized data
 }
 
+/*
+ * The below variables(name, value) will be connected to getCompletions in paragraph.controller.js
+ *
+ * name: which is shown in the suggestion list
+ * value: actual return value what you selected
+ */
+struct InterpreterCompletion {
+  1: string name,
+  2: string value
+}
+
```

Comment:
@bzz I added some descriptions instead of changing overall variable names. Since `name` & `value` is already existed  variable name in `paragraph.controll.js`. What do you think? :)